pkg/testutil/testutil.go

```diff
@@ -69,3 +69,18 @@ func MakeTestContextDexEnabled() context.Context {
 	}))
 	return ctx
 }
+
+func MakeTestContextDexEnabledUser(user string) context.Context {
+	u := auth.User{
```

Comment:
either make `user` a pointer and default to "developer", or in the other function call this new function.

Reply:
Also please rename user to `role`

Reply:
Applied fix.